docs/guides/postinstall.md

```diff
@@ -142,6 +142,140 @@ blacklist cdc_mbim" >> /etc/modprobe.d/blacklist.conf'
 
 Please note that this internal ethernet interface is required for various services including touchid that there currently is no Linux support for. In the future, if any of these services are supported, you'll need to undo this.
 
+# How to use old touchbar driver
+
+Some users want to use `apple-touchbar` driver instead of `tiny-dfr`, here's how to do so:
+
+1. Create and edit the script file: `~/switch-old-touchbar.sh`
+
+2. Paste this content:
+
+    ```bash
+    #!/bin/bash
+    
+    # Checking if root
+    if [ "$EUID" -ne 0 ]
+      then echo "Please run as root"
+      exit 1
```

Comment:
You can just do sudo $0 here

Reply:
I added `sudo $0` and option to revert to the new driver